### DISCUSSION TITLE
Is the Create mod compatible with 1.20.1 Forge of Connector?

### DISCUSSION THREAD

**Discussion by TEsterk003** [Sinytra/Connector#1437] [Q&A] [answered]
Is the Create mod compatible with 1.20.1 Forge of Connector?

**Comment by WitherTM** [comment, marked as answer]
have you tried it? but I do think that the forge version is compatible

**Reply by WitherTM** [reply]
like I said, it works with the **forge** version of Create, you are using the **fabric** version of Create

**Reply by TEsterk003** [reply]
Both the Fabric and Forge versions aren’t working...

**Reply by coco875** [reply]
remove porting_lib

**Reply by coco875** [reply]
while using the forge version of create

**Reply by TEsterk003** [reply]
I'm not using Porting Lib.

**Reply by WitherTM** [reply]
> I'm not using Porting Lib.

send your modlist

**Reply by coco875** [reply]
then what error/crash report/lastest.log you get ?

**Reply by WitherTM** [reply]
never thought I would have to say this again, but remove Rubidium and replace it with Embeddium

**Reply by TEsterk003** [reply]
I did what you said, but it still crashes.

**Reply by WitherTM** [reply]
> I did what you said, but it still crashes.

try disabling ReplayMod

**Reply by TEsterk003** [reply]
still crashes

**Reply by WitherTM** [reply]
give us the log

**Reply by TEsterk003** [reply]
Guys, we’re close to a solution... Since it keeps crashing no matter which mod I remove, I figured, okay, let’s see if it works with just Create and a few others... and it does! But now I need to FIND WHAT'S CAUSING THE CRASH, meaning the mod that’s messing things up! Because I obviously need this pack to work with the other mods too.

**Reply by WitherTM** [reply]
> Guys, we’re close to a solution... Since it keeps crashing no matter which mod I remove, I figured, okay, let’s see if it works with just Create and a few others... and it does! But now I need to FIND WHAT'S CAUSING THE CRASH, meaning the mod that’s messing things up! Because I obviously need this pack to work with the other mods too.

could you send your latest.log or a crash log?

**Comment by Tapacywka** [comment]
Use Forge version instead.

**Reply by TEsterk003** [reply]
Both the Fabric and Forge versions aren’t working...

**Comment by Tapacywka** [comment]
Replace ReplayMod with the ReforgedPlay (its forge fork).

**Reply by TEsterk003** [reply]
And can i use connector with the multiplayer? (i only tested in singleplayer)

**Reply by Tapacywka** [reply]
Yes, you can use it on your server or via some mods like e4mc, World Host or Essentials.

**Comment by Su5eD** [comment]
Connector is compatible with all forge mods.
